net-analyzer/linkchecker/linkchecker-10.0.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+PYTHON_REQ_USE="sqlite?"
+
+inherit bash-completion-r1 distutils-r1 optfeature
+
+DESCRIPTION="Check websites for broken links"
+HOMEPAGE="https://github.com/linkcheck/linkchecker"
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/linkcheck/linkchecker.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/linkchecker/linkchecker/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="sqlite"
+# requires libs not present in portage yet
+RESTRICT="test"
+
+RDEPEND="
+	dev-python/beautifulsoup[${PYTHON_USEDEP}]
+	dev-python/dnspython[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+"
+
+python_prepare_all() {
+	local PATCHES=(
```

Comment:
can `local PATCHES` & `local DOCS` be moved into global scope? would allow removing `python_prepare_all()`

Reply:
Previously it was present in the global scope, but it has been noted to move it to local scope instead.

Reply:
Really? Why?

Reply:
To avoid polluting the global scope. However, it can be changed, I don't see a problem in that.

@thesamesam are you too suggesting to switching to global scope?

Reply:
Moved to global scope.